libs/src/services/creatorNode/index.js

```diff
@@ -67,6 +76,7 @@ class CreatorNode {
 
   constructor (web3Manager, creatorNodeEndpoint, isServer, userStateManager, lazyConnect, schemas) {
     this.web3Manager = web3Manager
+    // This is just 1 endpoint (primary), unlike the creator_node_endpoint field in user metadata
```

Comment:
i think it might be better to rename this to `primaryContentNodeEndpoint` or something but non-blocking

Reply:
agreed, but don't want to delay this PR on that change